### PR TITLE
Add `values` function for getting values from a `Map`

### PR DESCRIPTION
**Pull request by jdidion**
I also added the `Array[String] keys(Struct|Object)` function variant for getting the member names for a struct or object. I think this is useful, but also necessary to be consistent with the signatures of the `contains_key` function.

Also fixed the example for `contains_key`.

Also added the missing entry for multi-line strings to the changelog.

### Checklist

- [x] Pull request details were added to CHANGELOG.md
- [x] Valid examples WDL's were added or updated to the SPEC.md (see the [guide](https://github.com/openwdl/wdl-tests/blob/main/docs/MarkdownTests.md) on writing markdown tests)

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -9883,15 +9884,20 @@ Test config:
 
 ```
 Array[P] keys(Map[P, Y])
+Array[String] keys(Struct|Object)
 ```
 
-Creates an `Array` of the keys from the input `Map`, in the same order as the elements in the map.
+Given a key-value type collection (`Map`, `Struct`, or `Object`), returns an `Array` of the keys from the input collection, in the same order as the elements in the collection.
```
Comment (markjschreiber) [inline SPEC.md:10001]: If the key of the input is optional then does the returned Array contain a concrete String?

For example:

```
Struct Name {
    last: String,
    first: String,
    middle: String?
}

...
Name name = {
    first: "John",
    last: "Doe"
}

Array[String] the_keys = keys(name).
```
Does `the_keys` contain the `String middle` or not?

**Review by markjschreiber — COMMENTED** [review]
How should optional keys be handled when they are not set?

File: SPEC.md (hunk 1 above)
Comment (jdidion) [reply SPEC.md:10001]: Yes, for a struct all members are always present, and optional members default to `None` if a value is not specified.

**Review by markjschreiber — APPROVED** [review]
LGTM

**Pull request merged**